Userland/Libraries/LibChess/UCICommand.cpp

```diff
@@ -323,16 +324,181 @@ ErrorOr<String> BestMoveCommand::to_string() const
     return builder.to_string();
 }
 
-ErrorOr<NonnullOwnPtr<InfoCommand>> InfoCommand::from_string([[maybe_unused]] StringView command)
+ErrorOr<NonnullOwnPtr<InfoCommand>> InfoCommand::from_string(StringView command)
 {
-    // FIXME: Implement this.
-    VERIFY_NOT_REACHED();
+    auto tokens = command.split_view(' ');
+    VERIFY(tokens[0] == "info");
+
+    auto info_command = TRY(adopt_nonnull_own_or_enomem(new (nothrow) InfoCommand));
+
+    auto verify_integer_token = [](StringView value_token) {
```

Comment:
Two things with this lambda:
	- `VERIFY(value_as_integer.has_value());` will panic the program if `value_as_integer` doesn't have a value, exactly like `value_as_integer.value();`
	- Do we really want to crash if the command has wrong parameters? We can probably just discard it

Reply:
I've changed this to return an error, rather than panicking.